models/organization/src/team/types.ts

```diff
@@ -0,0 +1,25 @@
+import type { TBuilder } from '@commercetools-test-data/core';
+
+export type TTeam = {
+  id: string;
+  name: string;
+  members: []; // should be array of users. TODO: update when user model is ready
```

Comment:
We need to differentiate here between Rest models (`members` is a list of references) and GraphQL models (`members` is a list of CoCo Users).